### DISCUSSION TITLE
Outcomes/Success of support

### DISCUSSION THREAD

**Discussion by cmwyndham** [UNMigration/HTCDS#21] [Ideas]
Can we add 'outcomes' of the support (ie, when we 'close' a case, what is the outcome that we record) as a standard. This is a very complex area, but there are a number of sources based on research about survivor defined 'outcomes', including from IJM, RTI and the 'Self-Efficacy scale'. 

Definitely we need to base it on how survivors themselves define an outcome or a success.   

It's very tricky, and maybe too difficult to include, but it's also a need for us in terms of M&E - we need to somehow record our outcomes or success.

**Comment by philhb** [comment]
I agree - I think it's important and the whole purpose of case management! It's definitely tricky and lots of different opinions on this topic. Any other thoughts and opinions from the community on this are welcome!
